src/Links/__tests__/Link.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render } from 'test-utils';
+import { spy } from 'sinon';
+import { MemoryRouter } from 'react-router-dom';
+import { strictEqual } from 'assert';
+import Link from '../Link';
+
+describe('Link', function () {
+  let getByText;
+  let clickSpy;
```

Comment:
Since we don't have `onClick` anymore, we shouldn't need this click spy any more either

Reply:
Sorry about that. Fixed through 5b068f2